docs/environments/search_and_rescue.md

```diff
@@ -0,0 +1,54 @@
+# 🚁 Search & Rescue
+
+[//]: # (TODO: Add animated plot)
+
+Multi-agent environment, modelling a group of agents searching the environment
+for multiple targets. Agents are individually rewarded for finding a target
+that has not previously been detected.
+
+Each agent visualises a local region around it, creating a simple segmented view
+of locations of other agents in the vicinity. The environment is updated in the
+following sequence:
+
+- The velocity of searching agents are updated, and consequently their positions.
+- The positions of targets are updated.
+- Agents are rewarded for being within a fixed range of targets, and the target
+  being within its view cone.
+- Targets within detection range and an agents view cone are marked as found.
+- Local views of the environment are generated for each search agent.
+
+The agents are allotted a fixed number of steps to locate the targets. The search
+space is a uniform space with unit dimensions, and wrapped at the boundaries.
+
+## Observations
+
+- `searcher_views`: jax array (float) of shape `(num_searchers, num_vision)`. Each agent
+  generates an independent observation, an array of values representing the distance
+  along a ray from the agent to the nearest neighbour, with  each cell representing a
+  ray angle (with `num_vision` rays evenly distributed over the agents field of vision).
+  For example if an agent sees another agent straight ahead and `num_vision = 5` then
+  the observation array could be
+
+  ```
+  [1.0, 1.0, 0.5, 1.0, 1.0]
+  ```
+
+  where `1.0` indicates there is no agents along that ray, and `0.5` is the normalised
```

Comment:
I feel either 0 or -1 should indicate there is no agent along the ray, what do you think? Feels more natural to me.

Although I guess 1 indicates out of range, so 0 would be right in front. So maybe -1 is best...not sure what you think?

Reply:
Yeah my thinking was that 1.0 means objects/agents move away they do so continuosly until they are out of range at 0, felt the jump to -1 would maybe introduce slightly unatural discontinuity?

Reply:
Switched this over to set default as -1